BitwardenShared/UI/Vault/Vault/VaultItemSelection/VaultItemSelectionProcessor.swift

```diff
@@ -102,9 +109,30 @@ class VaultItemSelectionProcessor: StateProcessor<
             state.toast = newValue
         }
     }
+}
 
+extension VaultItemSelectionProcessor {
```

Comment:
⚠️  All the new code in this `extension VaultItemSelectionProcessor` is pretty much the same as in `VaultListProcessor` and `VaultGroupProcessor`. Could all of this be moved to another place where it can be shared amongst all the processors? Like a helper or some service so we improve maintainability, don't repeat unit tests across processors and we'd have shorter files.

Reply:
Good suggestion, I refactored this into a new `VaultItemMoreOptionsHelper`.